css/style.css

```diff
@@ -0,0 +1,198 @@
+* {
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    background-color: #ffffff;
+    font-family: 'Amatic SC' !important;
+}
+
+.aboutcolor{
+	background-color: #c7d8c6;
+
+}
+.sabrina {
+    height: 300px;
+    width: 300px;
+    margin: auto;
+    border: 20px solid white;
+    border-radius: 10px;
+}
+
+.intro {
+    font-family: 'Bungee Shade', cursive;
+    background-color: #fa7c92;
+    text-align: center;
+    padding-bottom: 20px;
+    padding-top: 20px;
+
+}
+
+.para {
+    font-family: 'Amatic SC', cursive;
```

Comment:
If you've assigned this font in your <body>, you shouldn't have to assign it again to specific tags.